themes/StarterTheme/templates/catalog/products.tpl

```diff
@@ -1,17 +1,21 @@
-{if $products|count}
-  <section id="products">
-    <h1>{l s='Products'}</h1>
+<section id="products">
+  {if $products|count}
+      <h1>{l s='Products'}</h1>
 
-    {block name="sort_by"}
-      {include './_partials/sort-by.tpl' options=$sort_options}
-    {/block}
+      {block name="sort_by"}
+        {include './_partials/sort-orders.tpl' sort_orders=$sort_orders}
+      {/block}
 
-    <div class="products">
-      {foreach from=$products item="product"}
-        {block name="product_miniature"}
-          {include './product-miniature.tpl' product=$product}
-        {/block}
-      {/foreach}
-    </div>
-  </section>
-{/if}
+      <div class="products">
```

Comment:
Products on a category page should be in a list item format. Reference the issue I posted to schemaorg/schemaorg#605 and related to Plain Old Semantic HTML + Microdata Formats.